src/components/wallet/card/wallet-card-colors.ts

```diff
@@ -0,0 +1,181 @@
+type TWalletCardColors = {
+    [key: string]: {
+        colors: {
+            primary: string;
+            secondary: string;
+        };
+    };
+};
+
+export const wallet_card_colors: TWalletCardColors = {
+    '1ForYou': {
+        colors: { primary: '#FF5F00', secondary: '#FF9900' },
+    },
+    Advcash: {
+        colors: { primary: '#00AF7E', secondary: '#1EE887' },
+    },
+    Airtm: {
+        colors: { primary: '#0083FF', secondary: '#00D1FF' },
+    },
+    ApplePay: {
+        colors: { primary: '#CBD1D8', secondary: '#E8EDF2' },
+    },
+    AstroPay: {
+        colors: { primary: '#FF0000', secondary: '#FF0743' },
+    },
+    AUD: {
+        colors: { primary: '#0DB43D', secondary: '#FFCD00' },
+    },
+    Banxa: {
+        colors: { primary: '#23BDAE', secondary: '#147ABB' },
+    },
+    Beyonic: {
+        colors: { primary: '#004361', secondary: '#B6CDED' },
+    },
+    Bitcoin: {
+        colors: { primary: '#F7931B', secondary: '#F7C71B' },
+    },
+    Boleto: {
+        colors: { primary: '#333333', secondary: '#8F8F8F' },
+    },
+    Changelly: {
+        colors: { primary: '#10D078', secondary: '#0BF8A3' },
+    },
+    Demo: {
+        colors: { primary: '#FF6444', secondary: '#FF444F' },
+    },
+    'Deriv P2P': {
+        colors: { primary: '#FF444F', secondary: '#FF6444' },
+    },
+    'Diners Club': {
+        colors: { primary: '#184977', secondary: '#228ECB' },
+    },
+    Directa24: {
+        colors: { primary: '#98D000', secondary: '#EAFF68' },
+    },
+    Dragonpay: {
+        colors: { primary: '#FFCE0E', secondary: '#BB1A20' },
+    },
+    'Dragon Phoenix': {
```

Comment:
Are we planning to access these object keys with spaces?
like this: wallet_card_colors[ 'Dragon Phoenix' ]

Reply:
@matin-binary only if we want to add some logics for this particular wallet inside the component. Currently, this naming with spaces implies that anyone can pass the wallet_name='Dragon Phoenix' props to the WalletCard component and the color scheme for this wallet will apply according to its respective colors in the `wallet_card_colors` object. I've removed the 'Dragon Phoenix' from this object because we are not going to implement a wallet for this payment method.